RELEASE_NOTES.md

```diff
@@ -48,6 +46,22 @@ Kitchen-ec2 has been updated to 3.0, which uses the newer `aws-sdk-v3` and inclu
 
 Chef Provisioning is no longer included with Chef DK, and will be officially end of life on August 31, 2019. The source code of Chef Provisioning and the drivers have been moved into the chef-boneyard GitHub organization and will not be further maintained. Current users of Chef Provisioning should contact your Chef Customer Success Manager or Account Representative to review your options.
 
+### `knife bootstrap` against cloud providers
+
+`knife bootstrap` was [rewritten](https://github.com/chef/chef/blob/cfbb01cb5648297835941679bc9638d3a823ad5e/RELEASE_NOTES.md#knife-bootstrap) in Chef Infra Client 15.
+in Chef Infra Client 15. The `knife-*` cloud providers need to be updated to use this new API. As of ChefDK 4.0 `knife
```

Comment:
```suggestion
The `knife-*` cloud providers need to be updated to use this new API. As of ChefDK 4.0 `knife
```

Reply:
This phrase was just orphaned, unless it was supposed to be a part of the sentence? If so, then the period will need to be changed to a comma.

Reply:
Actually, two phrases were orphaned.

Reply:
Ugh, sorry - merge issue. Let me back up and make sure I don't do that merge conflict.